playbooks/incident-handling.md

```diff
@@ -315,6 +315,8 @@ Many people follow the #incidents channel closely, so avoid noise by keeping dis
   <!-- TODO: define how to apply priority levels more consistently -->
 - Take your best guess at the other information; it can always be edited later.
 
+Issues isolated to staging (i.e., issues that do not have have an observable impact on production) resulting in major workflow disruptions should also be treated as incidents. When communicating a staging incident, it should be made clear that there is no preceived impact on production to avoid creating confusion for the team members following the incident.
```

Comment:
"Staging" is just one of our development tools, and I think this could apply equally to other tools (e.g. jenkins), so I'd consider describing these more generally as "non-production" disruptions that significantly affect teams' ability to function.

I would also recommend explicitly recommending the "low" or "informational" severity level for these.